.github/workflows/tests.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Tests
+
+on:
+  push:
+    branches:
+    # Check all branches
+  pull_request:
+    # Check all PR
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        python-version: [3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Lint with flake8
+      run: |
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
In the style guide of MDTools I allowed a maximum line length of 72 characters for code and docstings. The [PEP8 ](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) style guide allows a line length of 79 characters for code and 72 for docstrings. But these line lengths are not set in stone and can be extended if this improves the readability.

I'm not sure how we can handle exceptionally longer lines. Maybe we just set `--max-line-length` to 79. In this way developers should limit their lines to 72 characters, but might exceptionally go up to 79 characters, which will be the absolute limit.

Reply:
Set to line length 79, but this generates only a warning. So one _could_ have longer lines in some _exceptional_ cases without breaking anything.